### PR TITLE
MNT: Remove TEAL support

### LINKED ISSUE #106: MNT: Remove TEAL
If TEAL interface is not used here, we can safely remove it. @tddesjardins will discuss with @nagrogin

### PR DESCRIPTION
**Pull request by pllim**
Fix #106 

**TODO**

- [x] Get approval from @tddesjardins

### CODE REVIEW

File: acstools/acs_destripe_plus.py [hunk 1]
```diff
@@ -18,20 +18,14 @@
 Examples
 --------
 
-In Python without TEAL (recommended):
+In Python (recommended):
```
Comment (nmiles2718) [inline acstools/acs_destripe_plus.py]: Do we need the "(recommended)" part? With the removal of TEAL, there is no other options from within acstools and so this is redundant.

File: acstools/acs_destripe.py [hunk 2]
```diff
@@ -22,19 +22,13 @@
 Examples
 --------
 
-In Python without TEAL (recommended):
+In Python (recommended):
```
Comment (nmiles2718) [inline acstools/acs_destripe.py]: Again, no need to include "(recommended)" as using it via Python is the only option

File: acstools/acs_destripe_plus.py (hunk 1 above)
Comment (pllim) [reply acstools/acs_destripe_plus.py]: I think we do because there is also the CLI option?

File: acstools/acs_destripe.py (hunk 2 above)
Comment (pllim) [reply acstools/acs_destripe.py]: Same as above. It has a CLI.

File: acstools/acs_destripe_plus.py (hunk 1 above)
Comment (nmiles2718) [reply acstools/acs_destripe_plus.py]: We only ever mention running `calacs.e` on the RTD pages and the ACS DataHandbook always references the `acstools` interface. If you're tied to it, I'm fine with leaving it but it just seems weird to say something is recommended but never mention the other option that its recommended over.

Comment (pllim) [reply acstools/acs_destripe_plus.py]: Haha... okie dokie. Good point. I'll remove it.

**Comment by pllim** [comment]
@nmiles2718 , if it looks good to you now, please approve. Thanks!

**Comment by pllim** [comment]
Jenkins failure is due to Artifactory being down and not related to this PR.

Thanks for the review!

**Pull request merged**